message_tagging_service/config.py

```diff
@@ -40,7 +40,9 @@ class Config:
     Setting a new value to instance['name'] will override the value of property
         (but only for this particular instance)
     """
-    _defaults = {}
+    _defaults = {
+        'build_state_msg_filter': ['ready', 'done']
```

Comment:
Is this needed if they're set in the `BaseConfiguration` class?

Reply:
Seems no IIRC. `_defaults` could have configs that are not defined in the config classes to provide the default values.

Reply:
Yeah, I assume that the default value should be `['ready', 'done']` even it's not configured in `*Configuration`. Is it better to be removed?